src/Transformers/FamilyTransformer.php

```diff
@@ -21,7 +20,7 @@ public function transform(Entities\Family $family)
                 'weekStartDay'        => $this->transformWeekStartDay($family->getWeekStartDay()),
             ],
             'links'      => [
```

Comment:
Fractal will take care of links for you too, btw.

Reply:
![](https://media.giphy.com/media/yJFeycRK2DB4c/giphy.gif)